test/unit_Zonotope.jl

```diff
@@ -97,3 +97,19 @@ for N in [Float64, Rational{Int}, Float32]
     @test Z.center == N[2, 3] && diag(Z.generators) == N[4, 5]
     convert(Zonotope, BallInf(N[5, 3], N(2)))
 end
+
+for N in [Float64, Rational{Int}]
+    # conversion to HPolytope
+    # 1D
+    Z = Zonotope(N[0], Matrix{N}(I, 1, 1))
+    P = HPolytope(constraints_list(Z))
+    for d in [N[1], N[-1]]
+        @test ρ(d, P) == ρ(d, Z)
+    end
+    # 2D
+    Z = Zonotope(N[0, 0], Matrix{N}(I, 2, 2))
+    P = HPolytope(constraints_list(Z))
+    for d in [N[1, 0], N[-1, 0], N[0, 1], N[0, -1], N[1, 1]]
```

Comment:
more succintly `LazySets.Approximations.BoxDirections(2))` or ``LazySets.Approximations.BoxDiagDirections(2))``

Reply:
It actually has the same number of characters :smile:

Reply:
don't tell me that you counted them 📐

Reply:
No, you can see if [in the diff](https://github.com/JuliaReach/LazySets.jl/pull/958/commits/d97e8dc1f81cc23243f8240cc223be9d48de7da3) :roll_eyes: